### ISSUE TITLE
How do I use the packages installed once inside the chroot folder?

### ISSUE THREAD

**Issue by thebeardedgeek** [IBM/ibmichroot#85]
I ran these commands:
`yum install ibmichroot`

`chroot_setup -y /QOpenSys/chRootRiby`

`yum -y --installroot=/QOpenSys/chRootRiby install yum ca-certificates-mozilla`
This installed 53 packages including yum

`chroot /QOpenSys/chRootRiby /QOpenSys/pkgs/bin/bash`
this took me into the chroot container to a bash session; however, when I try to run `yum list installed` I get this error

`bash: yum: command not found`

When I perform an ls of the chroot container I am in this is what I see
QOpenSys
dev
lib
tmp
var
bin
home
sbin
usr

![image](https://user-images.githubusercontent.com/9466407/187484998-a076c68c-31a5-4db8-8720-c7b916c7f34c.png)

What am I missing here?

**Comment by kadler** [comment]
What is your PATH set to in the chroot?

**Comment by thebeardedgeek** [comment]
@kadler I don't know what to set my PATH to. I am new in my understanding of how some of these things work.

normally I would set my PATH using this:
`echo 'PATH=/QOpenSys/pjgs/bin:$PATH' >> $HOME/.profile`
`echo 'export PATH' >> $HOME/.profile`

What would I set it to for the chroot packages?

**Comment by kadler** [comment]
You would do the same thing, but you need to do that in the chroot.

**Comment by thebeardedgeek** [comment]
So run this within the chroot?
`echo 'PATH=/QOpenSys/bin:$PATH' >> $HOME/.profile`
`echo 'export PATH' >> $HOME/.profile`

**Comment by thebeardedgeek** [comment]
Ok, I see. I set my PATH the same way but the chroot knows how to view the packages that are located in the "container".

I ran this: 
`echo 'PATH=/QOpenSys/pkgs/bin:$PATH' >> $HOME/.profile`
`echo 'export PATH' >> $HOME/.profile`
disconnected from my SSH session and signed back in.

entered the chroot using
`chroot /QOpenSys/chRootRiby /QOpenSys/pkgs/bin/bash`

and when I run `echo $PATH` I only see the packages installed in the chroot

This was a new system that I just setup so my PATH had not been configured for this user before.
